MAUI/TextInputLayout/Migration.md

```diff
@@ -0,0 +1,59 @@
+---
+layout: post
+title: Migrating from Xamarin to .NET MAUI SfTextInputLayout | Syncfusion 
+description: Learn all about Migrating from Syncfusion Xamarin TextInputLayout to Syncfusion .NET MAUI TextInputLayout control and more here.
+platform: maui
+control: SfTextInputLayout
+documentation: ug
+---  
+
+# Migrating from Xamarin TextInputLayout to .NET MAUI TextInputLayout 
```

Comment:
# Migrating from Xamarin TextInputLayout to .NET MAUI TextInputLayout.

Reply:
Addressed the changes